src/core/utils.js

```diff
@@ -949,3 +950,54 @@ export class CaseInsensitiveMap extends Map {
     return super.delete(key.toLowerCase());
   }
 }
+
+export class Err extends Error {
+  /**
+   * @param {string} message
+   * @param {string} plugin Name of plugin that caused the error.
+   * @param {object} [options]
+   * @param {boolean} [options.isWarning]
+   * @param {string} [options.title]
```

Comment:
"title" is Kida weird... 

```suggestion
   * @param {string} [options.message]
```

Reply:
`title` is what gets applied to offending elements. similar to what we had in `showInlineError`. We already have message as a required constructor parameter.

Reply:
ok, makes sense... maybe worth a comment so I remember next time :)